generated/nidigitalpattern/nidigitalpattern_service.cpp

```diff
@@ -1760,7 +1761,7 @@ namespace nidigitalpattern_grpc {
       
         std::string value;
         if (buffer_size > 0) {
```

Comment:
Note this is a case where we need both the actual buffer_size and the C++ string size expression. So the codegen differentiates those  2.